ha_lineairdb.cc

```diff
@@ -838,6 +852,13 @@ void ha_lineairdb::set_write_buffer() {
 
     write_buffer_.append('"');
     for (; p < end; p++) write_buffer_.append(*p);
+    if ((*field)->is_nullable()) {
+      if (byte_nullflag[nullable_field_index] == 1) write_buffer_.append("NULL");
```

Comment:
What does it mean to add the string "NULL" ?
If you treat it as a magic number meaning null, can we distinguish between the string "NULL" and null?

Reply:
MySQL will not be able to distinguish "NULL" from NULL.
I cannot come up with a good way to express NULL.
Does '\N' work?
https://dev.mysql.com/doc/refman/8.0/en/null-values.html

Reply:
I think we need to change values to be stored in LineairDB (we are currently working at the part of extracting values from LineairDB, but that's not enough). 
Maybe a bitmap of `null_flags` could be stored in LineairDB.
IMO, a change in `write_row` and `update_row` will be nice.

Reply:
I believe that the output of Read in LineairDB is like "A","B","C", ...
How about we store the value like "flag-[01]-value-A","flag-[01]-value-B" ... or simply "[01]-A","[01]-B" or even "[01]A" ... ?

Reply:
The third option is to wrap values in `""` only for strings.
For example, NULL would be `""` and an empty string would be `""""`. This way allows to distinguish NULL and empty string, but the parsing logic would be difficult.

I recommend the second approach of providing headers because it is more flexible for future extensions.

Reply:
Yes, that would work fine.
Or you can create a header by adding a null-bitmap to the beginning of LineairDB values.
Then the layout of a value become like `[011],”", “1”, “3”`. 

The third option is to wrap values in `""` only for strings.
For example, NULL would be `""` and an empty string would be `""""`. This way allows to distinguish NULL and empty string, but the parsing logic would be difficult.

I recommend the second approach of providing headers because it is more flexible for future extensions.